dpe/src/bitmap.rs

```diff
@@ -0,0 +1,78 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+Abstract:
+    Helper struct for controlling and viewing bitmaps.
+--*/
+
+#[repr(C)]
+#[derive(Debug, PartialEq, Eq, Default)]
+pub struct Bitmap(u32);
+
+impl Bitmap {
+    /// Empty flags.
+    pub const fn const_default() -> Bitmap {
+        Bitmap(0)
+    }
+
+    /// Bitmap with only one flag set.
+    pub const fn flag(flag: usize) -> Bitmap {
```

Comment:
We might want to implement `BitOr` and `BitOr<usize>` to be able to write:
- `Bitmap::flag(FOO_FLAG) | Bitmap::flag(BAR_FLAG)`
- `Bitmap::default() | FOO_FLAG | BAR_FLAG`
- `Bitmap::flag(FOO_FLAG) | BAR_FLAG`